Tests/Marketplace/marketplace_services.py

```diff
@@ -565,6 +568,92 @@ def upload_to_storage(self, zip_pack_path, latest_version, storage_bucket, overr
             print_error(f"Failed in uploading {self._pack_name} pack to gcs. Additional info:\n {e}")
             return task_status, True
 
+    @staticmethod
+    def major_minor_rev(version, path=False):
+        """
+        Returns version string as int to be used for version comparison.
+        Accepts both paths (where version numbers use '_' as separators), as well as strings where
+        version separators are '.'
+        :param version: string containing version number.
+        :param path: Bool to indicate if searching within a path or string
+        :return: Tuple of integers.
+        """
+        if path:
+            major, minor, rev = re.search('(\d+)_(\d+)_(\d+)', version).groups()
+        else:
+            major, minor, rev = re.search('(\d+)\.(\d+)\.(\d+)', version).groups()
+        return int(major), int(minor), int(rev)
+
+    def prepare_release_notes(self, storage_bucket):
+        """Need to implement the changelog.md parsing and changelog.json creation after design is
+        finalized.
+
+        """
+        task_status = False
+        try:
+            metadata_path = os.path.join(self._pack_path, Pack.METADATA)
```

Comment:
Can remove it... This already done in `load_user_metadata`. Use `self.current_version` property for it. If you need additional data, create new property.